### DISCUSSION TITLE
Some fields are not translated

### DISCUSSION THREAD

**Discussion by aircrescio** [woocommerce/woocommerce#50835] [Checkout Flow]
I have Woocommerce translated 100% in Italian but with Woocommerce Gutemberg blocks some text (not editable) are not translated. 
<img width="783" alt="Screenshot 2024-08-21 alle 12 24 09" src="https://github.com/user-attachments/assets/cbbd1748-cc10-4ca4-a127-d2d87fe47b1b">

**Comment by opr** [comment]
Hi, we intend to ship [a fix for this](https://github.com/woocommerce/woocommerce/pull/50892) in 9.2.3
